ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ip_security.schema.yml

```
@@ -144,6 +144,20 @@ keys:
               valid_values:
                 - transport
                 - tunnel
+            flow_parallelization:
+              type: bool
+              description: |-
+                Enable flow parallelization, when enabled multiple cores
+                are used to parallelize the IPsec encryption and decryption processing
```

We try to end all descriptions with a `.` (not completly unified everywhere yet)
```suggestion
                Enable flow parallelization, when enabled multiple cores
                are used to parallelize the IPsec encryption and decryption processing.
```